### DISCUSSION TITLE
BottomTabs for responsive websites

### DISCUSSION THREAD

**Discussion by iAbhinav** [shadcn-ui/ui#4556] [Ideas]
I feel a bottom-bar with tabs is required for almost all web apps which have a 'Home' for a user.

Having a side drawer is good, but a dedicated bottom-bar makes it easier to navigate.

I would want a bottom-bar which has 4 most used pages, along with a 'More' button that opens up a side drawer.

I takes a lot of time to create a perfect bottom-bar, that has issues like safe-area and scroll area glitches sorted.

**Comment by gantz-cc** [comment]
You could simply use the tab component for this use case. If you would like to have the icon and label vertically stacked you could do this as well by using the flex col utility. 

For the „more“ option simply place a button inside the tab bar component that triggers the drawer.
